### ISSUE TITLE
The `contemporary` style dots should separate after a `subsection`

### ISSUE THREAD

**Issue by paddyroddy** [moderncv/moderncv#207]
**Is your feature request related to a problem? Please describe.**

I have split my work section into three things, but with the `contemporary` style means I get the dotted years joined up, which doesn't really make sense in this context.

```tex
\section[\faBriefcase]{Experience}

\subsection{Full-Time}

\cventry{2024--Present}{}{}{}{}{}

\subsection{Internships}

\cventry{2024--Present}{}{}{}{}{}

\subsection{Part-Time}

\cventry{2024--Present}{}{}{}{}{}
```

<img width="292" alt="image" src="https://github.com/user-attachments/assets/5e91f194-e286-4826-9ff0-d26da0c1e4ba">

**Describe the solution you'd like**
I would like the `\subsection` to break to start a new dotted section

**Describe alternatives you've considered**
I could insert an empty `\section{}` with some `\vspace` to move it up, but this is a bit hacky.

**Comment by LLdaniel** [comment]
Hi @paddyroddy, 
I am not sure if I understand it correctly:
Do you want to have separate timelines per `\subsection`? This would help me think about an solution. Thanks in advance.

**Comment by paddyroddy** [comment]
> Hi @paddyroddy, I am not sure if I understand it correctly: Do you want to have separate timelines per `\subsection`? This would help me think about an solution. Thanks in advance.

Yep, exactly that @LLdaniel! Not sure how common my use case it, but for people who make career changes or have internships or similar, it doesn't necessarily make sense to have the years joined up - even if they follow the same chronological order. For now I have done something like this, which does the trick

```tex
\section[\faBriefcase]{Experience}

\subsection{Full-Time}

\cventry{}{}{}{}{}{}

\section{}\vspace*{-10mm}
\subsection{Internships}

\cventry{}{}{}{}{}{}
```

**Comment by LLdaniel** [comment]
ah, okay, I see now. Thanks for clarification and your example. I will classify that as an possible enhancement :+1:
